options/locale/locale_en-US.ini

```diff
@@ -170,7 +170,7 @@ string.desc = Z - A
 
 [error]
 occurred = An error occurred
-report_message = If you are sure this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
+report_message = If you believe that this is a Gitea bug, please search for issues on <a href="%[1]s" target="_blank">%[2]s</a> or open a new issue if necessary.
```

Comment:
Maybe it's better to change the key to avoid other languages rendering error?

Reply:
Hi, thanks for the feedback, but I'm really not sure as to how I can address this.

Reply:
He meant that outdated translations which do not have the arguments may error because we pass more arguments in than the translation accepts, but I think this is something the translation function could gracefully handle.

Too much arguments should be gracefully ignored, only too few arguments is a real problem.

Reply:
Actually, it may not even be an issue because as far as I understand, the change will invalidate all existing translations, so they should fallback to english until translated, and no further changes are necessary here.